torch/tensor.py

```diff
@@ -98,7 +99,7 @@ def __deepcopy__(self, _memo):
             return memo[self._cdata]
         new_storage = self.storage().__deepcopy__(_memo)
         new_tensor = self.new()
-        new_tensor.set_(new_storage, self.storage_offset(), self.size(), self.stride())
+        new_tensor.set_(new_storage, self.storage_offset(), torch.Size(self.size()), self.stride())
```

Comment:
you're right, this isn't necessary anymore (`size()` used to return a `LongStorage` for sparse tensors)